src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import functions.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello Hello! I'm Duke\n" + "What can I do for you?");
+        int i = 0;
+        String txt;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+        try {
+            i = DukeSave.load(tasks);
+        } catch (Exception m) {
+            System.out.println("An exception has occurred: " + m);
+        }
+
+        while (true) {
+            txt = in.nextLine();
+            try {
+                if (txt.equals("list")) {                        //CMD LIST
```

Comment:
"list", "done", "todo"....you might want to consider defining them as constants.